src/Conversions.cc

```diff
@@ -301,6 +301,14 @@ msgs::Material ignition::gazebo::convert(const sdf::Material &_in)
           asFullPath(workflow->EnvironmentMap(), _in.FilePath()));
       pbrMsg->set_emissive_map(workflow->EmissiveMap().empty() ? "" :
           asFullPath(workflow->EmissiveMap(), _in.FilePath()));
+
+      // todo(anyone) add light_map to material.proto
```

Comment:
Can we ticket an issue for this?

Reply:
ticketed ignitionrobotics/ign-msgs#109

Reply:
This was addressed on https://github.com/ignitionrobotics/ign-msgs/pull/111. We could wait until `ign-gazebo5` is updated to use `ign-msgs7` and use the new field here (https://github.com/ignition-tooling/release-tools/issues/362)

Reply:
sure sounds good

Reply:
I think this is unblocked now